docs/source/autoscaling.rst

```diff
@@ -27,9 +27,10 @@ Let's look at sample marathon config file:
      autoscaling:
        decision_policy: pid
        metrics_provider: mesos_cpu_ram
-       delay: 300
        setpoint: 0.5
 
+This makes the instance ``main`` autoscale using the ``pid`` decision policy and the ``mesos_cpu_ram`` metrics provider. Paasta will aim to keep this service's utilization at 50%.
```

Comment:
I know we have a few other instances where we say `Paasta`, but it is probably worth trying to be consistent and use `PaaSTA`.